### ISSUE TITLE
Allow unretrieved publications to be listed

### ISSUE THREAD

**Issue by jpmckinney** [open-contracting/data-registry#227]
We'll need to add logic to display a relevant, customizable message.

That way, people can search for a known publisher of contracting data, and read why we don't have their data (e.g. the JSON is invalid, the publication went offline before the registry launched, etc.).

**Comment by jpmckinney** [comment]
@yolile It might be worth doing this so that we can have a page for CCE (Colombia).

**Comment by yolile** [comment]
Sure, but do you remember why CCE is currently not listed?
And for 

> read why we don't have their data (e.g. the JSON is invalid, the publication went offline before the registry launched, etc.).

Should we add (and populate) a new field where we manually indicate the reason?

**Comment by jpmckinney** [comment]
Yes, we would need a new translatable field – maybe Markdown if we can link to an explanation.

**Comment by yolile** [comment]
Currently these publishers are not shown in the registry:

Afghanistan -> down before the registry
Colombia Compra Eficiente -> scraping
Croatia -> Malformed JSON
Ecuador Emergency data -> down before the registry (Actually, I think we can remove this one from the registry and collect since it is not coming back)
European Union: Digiwhist -> not started
France -> scraping
Honduras SEFIN/ONCAE -> scraping
Indonesia: Indonesia Corruption Watch (ICW) -> scraping
Indonesia: Bandung - City Government -> down before the registry
Italy Digiwhist -> not started
Mexico: Administración Pública Federal -> scraping
Nepal: Dhangadhi -> scraping
Nigeria: Osun State -> incomplete JSON
Panamá: Dirección General de Contrataciones Públicas -> scraping
Romania: Digiwhist -> not started
Tanzania: Zabuni -> down before the registry

So we have: 
- currently scraping (I guess we can automatize this, e.g., if the active job is in a running state or something similar)
- down before the registry and 
- incomplete or malformed JSON (and a few from digiwhist that we never started).

**Comment by yolile** [comment]
Ok, so are you happy with closing this issue for now? (or at least changing its label from priority to triage?)

**Comment by jpmckinney** [comment]
We can close, and re-open if there is new demand.

**Comment by yolile** [comment]
I will reopen this again as users have asked why México APF and Croatia are not there

**Comment by yolile** [comment]
This is also relevant for large datasets, which we are not able to process at the moment, like Prozorro (#304)

**Comment by yolile** [comment]
So, in order to implement this, we need to make the following changes:
1. We agreed on adding a new field to provide the rationale for a publication to be listed, but without data. How should we name this new field? `unretrieve_rationale` maybe?
2. Change the `CollectionQuerySet.visible` method to also return collections without active jobs but with an `unretrieve_rationale` set
3. Change the detail view to not show the Data available, Data quality, and Access Data section, and show the new field `unretrieve_rationale` if unretrieve_rationale is set

Anything else?

**Comment by yolile** [comment]
Agree, and for 3. I think we don't need to remove the sections as we already say the data is not available if there is no `job` for the collection, e.g.:

![Image](https://github.com/user-attachments/assets/917f4518-bdcb-42ce-86e9-aaa4ac1c4299)

**Comment by yolile** [comment]
After reviewing our MVP pipeline, we are missing:

- chile_public_works_bulk (currently not responding to requests)
- Comisión Estatal de Garantía de Acceso a la Información Pública del Estado de San Luis Potosí (CEGAIPSLP) (Mexico)
- mexico_administracion_publica_federal
- mexico_jalisco (deleted)
- mexico_cdmx (deleted)
- mexico_grupo_aeroporto (deleted)
- Mexico: Secretaría de Comunicaciones y Transportes (SCT)
- mexico_quintana_roo_sesaeqroo_plataforma_digital_nacional
- mexico_veracruz_state_sesea_plataforma_digital_nacional
- mexico_mexico_state_sesaemm_plataforma_digital_nacional
- mexico_michoacan_sesea_plataforma_digital_nacional
- mexico_puebla_state_seseap_plataforma_digital_nacional
- mexico_guadalajara
- moldova_old (obsoleted)
- moldova_positive_initiative (deleted)
- nicaragua_solid_waste (deleted)
- ukraine

From that list, I think it only makes sense to add `ukraine` and `mexico_administracion_publica_federal`

**Comment by yolile** [comment]
Ah, re: Mexico, neither the API nor bulk downloads are working now, so I guess only Ukraine it is.
